electron/games.ts

```diff
@@ -20,26 +19,32 @@ abstract class Game {
       return GOGGame.get(appName)
     }
   }
+  public get logFileLocation() {
+    return join(
+      heroicGamesConfigPath,
+      `${this.appName.split(' ')[0]}-lastPlay.log`
```

Comment:
```suggestion
      `${this.appName}-lastPlay.log`
```

Reply:
Well, this was how the filename was originally generated. But I do agree that the split is useless, AppNames can't have spaces